bake/bake.go

```diff
@@ -534,7 +535,8 @@ func (c Config) newOverrides(v []string) (map[string]map[string]Override, error)
 		}
 
 		pattern := keys[0]
-		if len(parts) != 2 && keys[1] != "args" {
+		attrName := strings.TrimSuffix(keys[1], "+")
```

Comment:
this doesn't look very logical to do multiple `TrimSuffix`. Instead there should be one `TrimSuffix` for `parts[0]`